google/cloud/spanner/samples/samples.cc

```diff
@@ -2500,6 +2500,10 @@ void UsePartitionQuery(google::cloud::spanner::Client client) {
   for (auto const& partition : *partitions) {
     auto rows = client.ExecuteQuery(partition);
     for (auto& row : spanner::StreamOf<RowType>(rows)) {
+      if (row.status().code() ==                           //! TODO(#11201)
```

Comment:
The `//!` comment is a sigil for Doxygen.  I am not sure what effect, if any, would this have in how the samples are rendered.

Reply:
Followed up offline to highlight that it should have an effect.  And while it will in this particular case, it also looks like the required support has not been applied to all tags.  I'll chase that down internally.  Thanks.